docs/source/getting-started/setting-up-hypervisor-driver-plugin.adoc

```diff
@@ -152,5 +151,36 @@ $ sudo chown root:wheel /usr/local/bin/docker-machine-driver-xhyve
 $ sudo chmod u+s /usr/local/bin/docker-machine-driver-xhyve
 ----
 
-For more information, refer to the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver] documentation on GitHub.
+For more information, see the https://github.com/zchee/docker-machine-driver-xhyve#install[xhyve driver] documentation on GitHub.
 ====
+
+== Hyper-V
```

Comment:
This whole PR was about finding a better place for Hyper-V information and this discussion has been going on for a week here and in #843